requirements/common.in

```diff
@@ -166,9 +166,8 @@ disposable-email-domains
 yamole
 
 # Needed for signing thumbnail requests so that they can be authenticated on the
-# other end.  Using a fork to eliminate a really slow pkgresources import:
-# https://github.com/thumbor/libthumbor/pull/37
-https://github.com/zulip/libthumbor/archive/60ed2431c07686a12f2770b2d852c5650f3ccfc6.zip#egg=libthumbor==1.3.2zulip
+# other end.
+libthumbor==2.0.1
```

Comment:
The version should not be pinned here.

Reply:
This is merged now. Can we unpin it next time we upgrade?

Reply:
The rule is simple: *do not pin versions in `*.in`*.

If you have a specific reason to pin a version in `*.in` that warrants an exception to this rule, you’ll know what it is. Every existing instance of this is commented.

Pinning versions in `*.in` does not increase the reproducibility of development or production installs—those only use the automatically generated `*.txt` lock files, which already have every version pinned. But it does prevent us from taking upgrades even when we want to take upgrades (since we do that by deleting `*.txt` and rerunning `update-locked-requirements`).

Reply:
oh! I understand now. Thanks. Opened #14699 to do so.

Reply:
Ooops! Looks like you already created one.